core/src/main/resources/com/squarespace/template/plugins/platform/quantity-input.html

```diff
@@ -1,6 +1,6 @@
 {.var @quantityLabel localizedStrings.productQuantityInputLabel}
 
 <div class="product-quantity-input" data-item-id="{id}" data-animation-role="content">
-  <div class="quantity-label">{.if @quantityLabel}{@quantityLabel}{.or}Quantity{.end}:</div>
-  <input size="4" max="9999" min="1" value="1" type="number" step="1"></input>
+  <div class="quantity-label" id="sqs-product-quantity-label">{.if @quantityLabel}{@quantityLabel}{.or}Quantity{.end}:</div>
```

Comment:
Unrelated to the PR: can "Quantity" be surfaced to international users untranslated?